src/app/core/footer/footer.component.css

```diff
@@ -22,3 +23,8 @@ footer > * {
 a {
   color:white;
 }
+
+.social-icon {
+  height: 1.5em;
+  width: 1.5em;
+}
```

Comment:
Same thing as the font, it seems too small.

Reply:
I'd say try out **2em** (with appropriate margins to make it not look weird.